### PR TITLE
don't clean it after build

### PR DESCRIPTION
**Pull request by adrpo**
### Related Issues

None

### Purpose

Don't clean after install as people might compile from zip not git repository

### Approach

Change to the Makefile.

### Type of Change

- build changes

### Checklist

- I have performed a self-review of my own code

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/483//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#276

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/483/)** The test suite is unstable. :unamused:

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :smiley:

File: Makefile [hunk 1]
```diff
@@ -222,7 +222,6 @@ config-libxml2:
 	@echo
 	$(MKDIR) 3rdParty/libxml2/$(INSTALL_DIR)
 	cd 3rdParty/libxml2 && ./autogen.sh --prefix="$(ROOT_DIR)/3rdParty/libxml2/$(INSTALL_DIR)" --without-python $(HOST_CROSS_TRIPLE) && $(MAKE) && $(MAKE) install
-	cd 3rdParty/libxml2 && git clean -fdx -e 'install/' || true
```
Comment (lochel) [inline Makefile]: @adrpo If this is not cleaned up, then it must use different directories for each target (mingw, mac, linux, ...).

Comment (lochel) [reply Makefile]: Cleaning it was actually just a workaround ...

**Comment by adrpo** [comment]
Ouch. Then we need some real clean target for libxml2

**Comment by adrpo** [comment]
I'll have a look tomorrow.

**Comment by lochel** [comment]
I think the autoconf.sh file doesn’t support different build directories. We could either modify that script or add a clean target.

**Pull request closed without merging**